content/en/flux/installation/_index.md

```diff
@@ -157,16 +157,91 @@ The provider offers a Terraform resource called
+Check out the examples available for the provider in the
+[fluxcd/terraform-provider-flux](https://github.com/fluxcd/terraform-provider-flux) repository.
 
-- [Bootstrapping a cluster using a GitHub repository and a personal access token (PAT)](https://github.com/fluxcd/terraform-provider-flux/tree/main/examples/github-via-pat)
-- [Bootstrapping a cluster using a GitHub repository via SSH](https://github.com/fluxcd/terraform-provider-flux/tree/main/examples/github-via-ssh)
-- [Bootstrapping a cluster using a GitHub repository via SSH and GPG](https://github.com/fluxcd/terraform-provider-flux/tree/main/examples/github-via-ssh-with-gpg)
-- [Bootstrapping a cluster using a GitHub repository self-managing the SSH keypair secret)](https://github.com/fluxcd/terraform-provider-flux/tree/main/examples/github-self-managed-ssh-keypair)
-- [Bootstrapping a cluster using a Gitlab repository via SSH](https://github.com/fluxcd/terraform-provider-flux/tree/main/examples/gitlab-via-ssh)
+### Bootstrap with Flux Operator
 
-For more details on how to use the Terraform provider
-please see the [Flux docs on registry.terraform.io](https://registry.terraform.io/providers/fluxcd/flux/latest/docs).
+The [Flux Operator](https://github.com/controlplaneio-fluxcd/flux-operator) is an open-source project
+part of the [Flux ecosystem](/ecosystem/#flux-extensions) that provides a declarative API for the
+lifecycle management of the Flux controllers.
+
+The operator offers an alternative to the Flux CLI bootstrap procedure, with the option to configure the
+reconciliation of the cluster state from Git repositories, OCI Artifacts, or S3-compatible storage.
+
+#### Install the Flux Operator
+
+Install the Flux Operator in the `flux-system` namespace, for example, using Helm:
+
+```shell
+helm install flux-operator oci://ghcr.io/controlplaneio-fluxcd/charts/flux-operator \
+  --namespace flux-system \
+  --create-namespace
+```
+
+The Flux Operator can be installed using Helm, Terraform, OpenTofu, OperatorHub, and other methods.
+For more information, refer to the [installation guide](https://fluxcd.control-plane.io/operator/install/).
+
+#### Configure the Flux Instance
+
+Create a [FluxInstance](https://fluxcd.control-plane.io/operator/fluxinstance/) resource
+named `flux` in the `flux-system` namespace to install the latest Flux stable version and configure the
+Flux controllers to sync the cluster state from an OCI artifact stored in GitHub Container Registry:
+
+```yaml
+apiVersion: fluxcd.controlplane.io/v1
+kind: FluxInstance
+metadata:
+  name: flux
+  namespace: flux-system
+  annotations:
+    fluxcd.controlplane.io/reconcileEvery: "1h"
+    fluxcd.controlplane.io/reconcileArtifactEvery: "10m"
+    fluxcd.controlplane.io/reconcileTimeout: "5m"
+spec:
+  distribution:
+    version: "2.x" # automatic upgrade to the latest Flux version
```

Comment:
I'd prefer this example to be for the latest minor release patches instead of the latest minor releases.

Reply:
This would mean updating the example for every flux release and we’ll forget about it. Also this sample matches what’s in the operator readme.

Reply:
I'm fine including this as an item in the website release procedure, there's already a place where we do it anyway (the website home page).

I think we should update the sample in the operator readme.

We should definitely do more than our best to avoid breakages across minor releases, but sometimes it'll just slip through the cracks.

Reply:
People should read the docs and decide what upgrade policy to use. I see no reason why would we complicate the website release procedure for this. Also this is an ecosystem project, other maintainers shouldn’t have to do more work because of this docs addition.

Reply:
added